website/public/locales/ro/side_menu.json

```diff
@@ -0,0 +1,12 @@
+{
+  "dashboard": "Panou de control",
```

Comment:
I believe this file was removed.  Can you move these entries into `common.json`?